src/app/components/EmailInput/EmailInput.types.ts

```diff
@@ -3,11 +3,11 @@ import { ChangeEvent } from "react";
 export interface EmailInputProps {
```

Comment:
Lets add few important attributes here: id, name. And verify them in tests

Reply:
Done! I made one test case for both atributes